qa/rpc-tests/test_framework/test_node.py

```diff
@@ -18,6 +18,7 @@ def _raise_assertion_error(self, error):
 
     @contextlib.contextmanager
     def assert_debug_log(self, *, expected_msgs, unexpected_msgs):
+        self.logline('flush log')
```

Comment:
This needs to be in the `finally` section, when we re-open the log file to read the differences between start and end.

Reply:
yes, thankyou!

Reply:
On second though, it needs to be in both places!

Reply:
yes again!